.docker/docker-compose-ocr.yml

```diff
@@ -72,7 +72,7 @@ services:
 
   form-rec-custom-api:
     container_name: azure-cognitive-service-custom-api
-    image: mcr.microsoft.com/azure-cognitive-services/form-recognizer/custom-api
+    image: mcr.microsoft.com/azure-cognitive-services/form-recognizer/custom-api:2.1-preview
```

Comment:
For some reason custom-api:latest no longer exposes the forms api in swagger, only the layout api. We need the forms api to be able to modify or create composite models.